### ISSUE TITLE
Durum Wheat Classification using DL

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#246]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Durum Wheat Classification</br>
:red_circle: **Aim** : Create a DL model which will identify the proper images of the Durum wheat.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/muratkokludataset/durum-wheat-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by riyalguha** [comment]
Full name : Riyal Guha
GitHub Profile Link : https://github.com/riyalguha
Email ID : riyalguha2002@gmail.com
Approach for this Project :
1)Collect a diverse dataset of Durum wheat images(from kaggle).
2)Preprocess the dataset by resizing, normalizing, and augmenting the images.(if needed).
3)Select a suitable deep learning architecture like VGG, ResNet, or Inception for image classification.
4)Train the model using the dataset, optimizing the chosen loss function and optimizer.
5)Evaluate the model's performance using a separate test dataset.
What is your participant role?: Contributor at SSOC season 2

**Comment by abhisheks008** [comment]
Hi @riyalguha nice to have you here. This issue will be assigned to you once the program starts officially ie. June 1.

Till then, star the repository and explore more and more in open source.

**Comment by abhisheks008** [comment]
Issue assigned to @riyalguha

**Comment by the-silent-geek** [comment]
Full name : D. Uday Kiran
GitHub Profile Link : https://github.com/the-silent-geek
Email ID : udaykir3911@gmail.com
Approach for this Project : This can be implemented by using 4-5 tansfer learning models.
What is your participant role? social winter of code 23

**Comment by abhisheks008** [comment]
Assigned under SWOC S4 @the-silent-geek
